packages/electron-updater/src/MacUpdater.ts

```diff
@@ -120,7 +119,8 @@ export class MacUpdater extends AppUpdater {
       },
       done: event => {
         try {
-          copyFileSync(event.downloadedFile, cachedUpdateFile)
+          const cachedUpdateFilePath = path.join(this.downloadedUpdateHelper!.cacheDir, CURRENT_MAC_APP_ZIP_FILE_NAME)
+          copyFileSync(event.downloadedFile, cachedUpdateFilePath)
```

Comment:
What are your thoughts on converting this to a `log.warn` since it's not a critical error and still allows the update to proceed? Also, I think we should check if `disableDifferentialDownload` is `false` since caching is unnecessary if differentials are disabled.
```
done: event => {
  if (!downloadUpdateOptions.disableDifferentialDownload) {
    try {
      const cachedUpdateFilePath = path.join(this.downloadedUpdateHelper!.cacheDir, CURRENT_MAC_APP_ZIP_FILE_NAME)
      copyFileSync(event.downloadedFile, cachedUpdateFilePath)
    } catch (error: any) {
      this._logger.warn(`Unable to copy file for caching for future differential downloads: ${error.message}`)
    }  
  }
  return this.updateDownloaded(zipFileInfo, event)
},
```

Reply:
Agreed. Fixed.